packages/gatsby-theme-twentytwenty/package.json

```diff
@@ -1,17 +1,20 @@
 {
-    "name": "gatsby-theme-twentytwenty",
-    "version": "0.1.0",
-    "description": "The Gatsby Theme - ported from Twenty Twenty WordPress theme.",
-    "main": "index.js",
-    "license": "MIT",
-    "devDependencies": {
-      "gatsby": "^2.1.23",
-      "react": "^16.8.3",
-      "react-dom": "^16.8.3"
-    },
-    "peerDependencies": {
-      "gatsby": "^2.1.23",
-      "react": "^16.8.3",
-      "react-dom": "^16.8.3"
-    }
+  "name": "gatsby-theme-twentytwenty",
+  "version": "0.1.0",
+  "description": "The Gatsby Theme - ported from Twenty Twenty WordPress theme.",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "gatsby": "^2.1.23",
+    "react": "^16.8.3",
+    "react-dom": "^16.8.3"
+  },
+  "peerDependencies": {
```

Comment:
can we add the latest gatsby, react, react-dom dependencies?

Reply:
We have not specified version when installing so this should be the latest version.